com.jetbrains.IntelliJ-IDEA-Community.yaml

```diff
@@ -101,3 +103,30 @@ modules:
         path: com.jetbrains.IntelliJ-IDEA-Community.desktop
       - type: file
         path: com.jetbrains.IntelliJ-IDEA-Community.metainfo.xml
+
+  - name: idea-arm64
+    only-arches:
+      - aarch64
+    buildsystem: simple
+    build-commands:
+      - install --directory --mode=0755 /app/idea-IC/
+      - tar --directory=/app/idea-IC/ --extract --file=ideaIC.tar.gz --gunzip --strip-components=1
+      - install -D --mode=0644 /app/idea-IC/bin/idea.svg /app/share/icons/hicolor/scalable/apps/com.jetbrains.IntelliJ-IDEA-Community.svg
+      - install -D --mode=0755 entrypoint.sh /app/bin/idea
+      - install -D --mode=0644 --target-directory=/app/share/applications/ com.jetbrains.IntelliJ-IDEA-Community.desktop
+      - install -D --mode=0644 --target-directory=/app/share/metainfo/ com.jetbrains.IntelliJ-IDEA-Community.metainfo.xml
+    sources:
+      - type: file
+        path: entrypoint.sh
+      - type: file
+        dest-filename: ideaIC.tar.gz
+        sha256: cbd25b7e80f9b0b16574b2baee674df596fd6bdafac3fc51ae12fb0ed2836eac
+        url: https://download.jetbrains.com/idea/ideaIC-2022.3.1-aarch64.tar.gz
```

Comment:
As far as I can tell, this will merge the content of both, `ideaIC-YYYY.X.Z-aarch64.tar.gz` and `ideaIC-YYYY.X.Z.tar.gz`, into `/app/idea-IC/`. Because there is really no build process for this `Flatpak` I guess it's better to actually do that merge without repeating the rest of the common steps for both.

Reply:
I use "only-arches" attribute to make sure that only one of the two steps is executed. I tested it on the generated build and it worked correctly.

Reply:
If you look at the build output, it only downloads of the the two packages.

https://buildbot.flathub.org/#/builders/14/builds/1454/steps/8/logs/stdio